superset/viz.py

```diff
@@ -1178,7 +1190,7 @@ def to_series(self, df, classed="", title_suffix=""):
             chart_data.append(d)
         return chart_data
 
-    def process_data(self, df, aggregate=False):
+    def process_data(self, df: pd.DataFrame, aggregate=False):
```

Comment:
Nit. Whilst you here could you type the `aggregate` argument and return type?

Reply:
done